clusters/main.tf

```diff
@@ -3,15 +3,12 @@ terraform {
   required_providers {
     rancher2 = {
       source  = "rancher/rancher2"
-      version = "1.13.0"
     }
     aws = {
       source  = "hashicorp/aws"
-      version = "~> 3.12.0"
     }
     template = {
       source  = "hashicorp/template"
-      version = "~> 2.2.0"
```

Comment:
Why remove these?

Reply:
We use the `clusters/.terraform.lock.hcl` file to lock in the provider versions so these were a bit redundant.